src/disco/pages/DiscoPane/index.js

```diff
@@ -149,33 +155,51 @@ export class DiscoPaneBase extends React.Component<InternalProps> {
     );
   }
 
+  getSupportURL() {
+    const { siteLang, match } = this.props;
+    const { platform, version } = match.params;
+
+    return `https://support.mozilla.org/1/firefox/${version}/${platform}/${siteLang}/personalized-addons`;
```

Comment:
just checking..did you hear back if this is the correct link or not?

Reply:
nope.. The link looks correct (correct url/title) but there is no content..